src/ape/api/accounts.py

```diff
@@ -135,7 +165,21 @@ def transfer(
 
         return self.call(txn, send_everything=value is None)
 
-    def deploy(self, contract: "ContractContainer", *args, **kwargs) -> "ContractInstance":
+    def deploy(self, contract: "ContractContainer", *args, **kwargs) -> ContractInstance:
```

Comment:
```suggestion
    def deploy(self, contract: "ContractContainer", *args, **kwargs) -> "ContractInstance":
```

^ This is why it is failing.
I gave some poor advice before asking to remove the quotes but turns out there was a different bug (that is now fixed. And we need the quotes. This is the reason for the failing tests!

Reply:
OH THANK YOU